com.zettlr.Zettlr.metainfo.xml

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>com.zettlr.Zettlr</id>
+  
+  <name>Zettlr</name>
+  <summary> A Markdown editor for the 21st century</summary>
+  
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-3.0-or-later</project_license>
+  <content_rating type="oars-1.1" />
+  <url type="homepage">https://www.zettlr.com</url>
+  
+  <description>
+    <p>
+      With Zettlr, writing professional texts is easy and motivating: Whether you are a college student, a researcher, a journalist, or an author — Zettlr has the right tools for you. Watch the video or continue reading to see what they are!
```

Comment:
Would be nice to expand this based on the website.

Reply:
Ditto here.

Reply:
Done!